EncodingStreamingHighAvailability/HighAvailability.Tests/E2ETests.cs

```diff
@@ -0,0 +1,179 @@
+            var mediaServiceInstanceFactory = new MediaServiceInstanceFactory(configService);
+
+            foreach (var config in configService.MediaServiceInstanceConfiguration)
+            {
+                var client = await mediaServiceInstanceFactory.GetMediaServiceInstanceAsync(config.Value.AccountName).ConfigureAwait(false);
+                client.LongRunningOperationRetryTimeout = 2;
+
+                await MediaServicesHelper.EnsureTransformExists(
+                    client,
+                    config.Value.ResourceGroup,
+                    config.Value.AccountName,
+                    transformName,
+                    new BuiltInStandardEncoderPreset(EncoderNamedPreset.AdaptiveStreaming)).ConfigureAwait(false);
+
+                await mediaServiceInstanceHealthService.CreateOrUpdateAsync(new MediaServiceInstanceHealthModel
+                {
+                    MediaServiceAccountName = config.Value.AccountName,
+                    HealthState = InstanceHealthState.Healthy,
+                    LastUpdated = DateTime.UtcNow,
+                    IsEnabled = true
+                },
+                    Mock.Of<ILogger>()).ConfigureAwait(false);
+
+                await MediaServicesHelper.EnsureContentKeyPolicyExists(
+                    client,
+                    config.Value.ResourceGroup,
+                    config.Value.AccountName,
+                    configService.ContentKeyPolicyName,
+                    configService.GetClearKeyStreamingKey(),
+                    configService.TokenIssuer,
+                    configService.TokenAudience).ConfigureAwait(false);
+            }
+
+            var target = new JobRequestStorageService(jobRequestQueue);
+            var uniqueness = Guid.NewGuid().ToString().Substring(0, 13);
+
+            for (var i = 0; i < 2; i++)
+            {
+                Assert.IsNotNull(await target.CreateAsync(GenerateJobRequestModel(i, uniqueness), Mock.Of<ILogger>()).ConfigureAwait(false));
+            }
+        }
+
+        /// <summary>
+        /// Generates test data
+        /// </summary>
+        /// <param name="sequenceNumber">sequence number</param>
+        /// <param name="uniqueness">unique part of the names</param>
+        /// <returns>Generated JobRequestModel</returns>
+        private static JobRequestModel GenerateJobRequestModel(int sequenceNumber, string uniqueness)
+        {
+            var jobId = $"jobId-{sequenceNumber}-{uniqueness}";
+            var jobName = $"jobName-{sequenceNumber}-{uniqueness}";
+            var outputAssetName = $"output-{sequenceNumber}-{uniqueness}";
+
+            // Add job input for this test
+            // TBD remove following value
+            var input = new JobInputHttp(
+                                   baseUri: "https://nimbuscdn-nimbuspm.streaming.mediaservices.windows.net/2b533311-b215-4409-80af-529c3e853622/",
```

Comment:
Is this just for the test sample? If so, we can keep it,but just wondering if this is covered in documentation to be changed.

Reply:
yep, it is part of setup documentation. I will remove it from here